packages/blip/app/components/messages/messageform.js

```diff
@@ -254,7 +258,11 @@ class MessageForm extends React.Component {
 
       if (canEditTimestamp) {
         editLink = (
-          <a className="messageform-change-datetime" href="" onClick={this.handleShowEditDate}>
+          <a
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Sementically, a "<a>" tag is incorrect. This should be a button. I suspect that the "<a>" tag has been originally chosen because its style looks like the one of a link. But it does not behave like a link (no navigation because of it, href empty, etc)

Reply:
100% agree with you. Is it ok if we review this when we migrate the notes logic to dumb? I feel like it would be complicated to do it now and much easier to do with MUI components once it's migrated

Reply:
The comment is non blocking :wink: